core/src/main/java/com/linecorp/armeria/common/metric/PrometheusVersion1MeterRegistries.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.metric;
+
+import static java.util.Objects.requireNonNull;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.prometheusmetrics.PrometheusConfig;
+import io.micrometer.prometheusmetrics.PrometheusMeterRegistry;
+import io.prometheus.metrics.model.registry.PrometheusRegistry;
+
+/**
+ * Provides the convenient factory methods for {@link PrometheusMeterRegistry}.
+ */
+@UnstableApi
+public final class PrometheusVersion1MeterRegistries {
```

Comment:
Optional) It is not easy to find this new class to use the new API.
How about removing `public static` methods and adding an accessor to `PrometheusMeterRegistries`?

For example:
```java
PrometheusMeterRegistries.V0.defaultRegistry()
PrometheusMeterRegistries.V1.defaultRegistry()
```

We may deprecate all static methods in `PrometheusMeterRegistries`.

Reply:
Same reason. :wink:
https://github.com/line/armeria/pull/5698#discussion_r1618040322